src/storage-preview/azext_storage_preview/commands.py

```diff
@@ -45,6 +45,30 @@ def get_custom_sdk(custom_module, client_factory, resource_type=CUSTOM_DATA_STOR
         g.custom_command('list', 'list_network_rules')
         g.custom_command('remove', 'remove_network_rule')
 
+    local_users_sdk = CliCommandType(
+        operations_tmpl='azext_storage_preview.vendored_sdks.azure_mgmt_storage.operations#'
+                        'LocalUsersOperations.{}',
+        client_factory=cf_local_users,
+        resource_type=CUSTOM_MGMT_STORAGE
+    )
+
+    local_users_custom_type = CliCommandType(
+        operations_tmpl='azext_storage_preview.operations.account#{}',
+        client_factory=cf_local_users,
+        resource_type=CUSTOM_MGMT_STORAGE
+    )
+
+    with self.command_group('storage account local-user', local_users_sdk,
+                            custom_command_type=local_users_custom_type,
+                            resource_type=CUSTOM_MGMT_STORAGE, min_api='2021-08-01', is_preview=True) as g:
+        g.storage_custom_command_oauth('create', 'create_local_user')
+        g.storage_custom_command_oauth('update', 'update_local_user')
+        g.storage_custom_command_oauth('delete', 'delete_local_user')
+        g.storage_custom_command_oauth('list', 'list_local_user')
+        g.storage_custom_command_oauth('show', 'show_local_user')
+        g.storage_custom_command_oauth('list-keys', 'list_keys_for_local_user')
+        g.storage_custom_command_oauth('regenerate-password', 'regenerate_password_for_local_user')
```

Comment:
For mgmt plane, we only use `g.command` and `g.custom_command`, no credential params needed